get-started/index.md

```diff
@@ -5,53 +5,24 @@ mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html
   - https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/index.html
 ---
+# Get started
 
-# Get started [elasticsearch-intro-what-is-es]
+{{es}} is the heart of the [Elastic Stack](the-stack.md). Combined with {{kib}}, it powers these Elastic solutions and use cases:
 
-[{{es}}](https://github.com/elastic/elasticsearch/) is a distributed search and analytics engine, scalable data store, and vector database built on Apache Lucene. It’s optimized for speed and relevance on production-scale workloads. Use {{es}} to search, index, store, and analyze data of all shapes and sizes in near real time.
+* [Observability](/solutions/observability.md)
+* [Security](/solutions/security.md)
+* [Search](/solutions/search.md)
 
-{{es}} is the heart of the [Elastic Stack](the-stack.md). Combined with [{{kib}}](https://www.elastic.co/kibana), it powers these Elastic solutions:
+:::{tip}
+Refer to our [customer success stories](https://www.elastic.co/customers/success-stories) for concrete examples of how Elastic is used in real-world scenarios.
+:::
 
-* [Observability](https://www.elastic.co/observability)
-* [Search](https://www.elastic.co/enterprise-search)
-* [Security](https://www.elastic.co/security)
+## First steps
```

Comment:
Consider something like "Next steps?"  "First steps" sounds like the user has a long way to go.  "Next steps" implies that the user is already making progress.